rdt/transformers/boolean.py

```diff
@@ -16,15 +16,16 @@ class BooleanTransformer(BaseTransformer):
     Null values are replaced using a ``NullTransformer``.
 
     Args:
-        nan (int or None):
-            Replace null values with the given value. If ``None``, do not replace them.
-            Defaults to ``-1``.
-        null_column (bool):
-            Whether to create a new column to indicate which values were null or not.
-            If ``None``, only create a new column when the fit data contains null values.
-            If ``True``, always create the new column whether there are null values or not.
-            If ``False``, do not create the new column.
+        missing_value_replacement (object or None):
+            Indicate what to do with the null values. If an object is given, replace them
+            with the given value. If the strings ``'mean'`` or ``'mode'`` are given, replace
```

Comment:
I guess we should remove `mean` since that doesn't really make sense for booleans